integreat_cms/cms/views/pages/page_bulk_actions.py

```diff
@@ -118,3 +121,62 @@ def post(self, request, *args, **kwargs):
 
         # Let the base view handle the redirect
         return super().post(request, *args, **kwargs)
+
+
+class ExportMultiLanguageXliffView(PageBulkActionMixin, BulkActionView):
+    """
+    Bulk action for generating XLIFF files for translations in multiple languages.
+    """
+
+    #: Whether the view requires change permissions
+    require_change_permission = False
+
+    def post(self, request, *args, **kwargs):
+        r"""
+        Function for handling a XLIFF export request for pages and multiple languages.
+        The pages get extracted from request.GET attribute and the request is forwarded to :func:`~integreat_cms.xliff.utils.pages_to_xliff_file`
+
+        :param request: The current request
+        :type request: ~django.http.HttpResponse
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The redirect
+        :rtype: ~django.http.HttpResponseRedirect
+        """
+
+        target_languages = [
+            get_object_or_404(
+                self.request.region.language_tree_nodes,
+                language__slug=language_slug,
+                parent__isnull=False,
+            ).language
+            for language_slug in self.request.POST.getlist("selected_language_ids[]")
```

Comment:
```suggestion
            language
            for language in self.request.region.active_languages
            if language.slug in self.request.POST.getlist("selected_language_ids[]")
```
This is a bit more complicated (and does more database queries) than needed.